.eslintrc.js

```diff
@@ -0,0 +1,39 @@
+module.exports = {
+	'env': {
+		'browser': true,
+		'es2021': true
+	},
+	'extends': [
+		'eslint:recommended',
+		'plugin:@typescript-eslint/recommended'
+	],
+	'overrides': [
+	],
+	'parser': '@typescript-eslint/parser',
+	'parserOptions': {
+		'ecmaVersion': 'latest',
+		'sourceType': 'module'
+	},
+	'plugins': [
+		'@typescript-eslint'
+	],
+	'rules': {
+		'indent': [
+			'error',
+			'tab'
+		],
+		'linebreak-style': [
+			'error',
+			'windows'
+		],
+		'quotes': [
+			'error',
+			'single'
+		],
+		'semi': [
+			'error',
+			'never'
+		],
+		'no-mixed-spaces-and-tabs': 0,

```

Comment:
skoro w innych miejsach masz `error` a nie `0` to utrzymaj spójność i też zostaw tutaj 'error'

Reply:
zmienione